### DISCUSSION TITLE
Icons in build-in font

### DISCUSSION THREAD

**Discussion by petr-nov** [HASwitchPlate/openHASP#777] [Q&A] [answered]
The documentation lists the icons that are available in the default font.

While searching for various example implementations of openhasp on the web, I came across that the icon with the code FA74, which is not mentioned in the documentation, also works. Is there a really complete list of icons available in the default font?

**Comment by petr-nov** [comment]
Nobody anything?

**Reply by fvanroie** [reply, marked as answer]
There were 12 new icons requested according to this file:
https://github.com/HASwitchPlate/openHASP/blob/master/src/font/md-icons.json
It would be great if someone can add them to the docs.

**Reply by fvanroie** [reply]
The icons list has been updated in the docs.

**Comment by abbottcam** [comment]
Hello, my new to OpenHasp, my WT32-SC01 Plus is running 0.7.0-rc12, I would like to install some new icons, specifically the transmission tower one, for export and import. How would I do this please?

Thanks

**Reply by fvanroie** [reply]
You can find the code points in the json file above:
```json
        "transmission-tower": "ED3E",
        "transmission-tower-export": "F92C",
        "transmission-tower-import": "F92D",
```
